FirstAPP/src/Main.kt

```diff
@@ -0,0 +1,4 @@
+fun main(args : Array<String>){
```

Comment:
i already delete this file but and made new commits but it didn't disappear here @nicolaiparlog